specification/src/chapter4.adoc

```diff
@@ -2,69 +2,897 @@
 
 == Use case examples
 
-=== Basic non-virtualized system
+This chapter provides guidance for building secure Risc-V systems for a selection of common use cases. It is not intended to be an exhaustive selection. The purpose is to illustrate common principles that can be applied in a wide variety of use cases. 
+
+Additional use cases may be added in future versions.
+
+=== Generic system
 
 ==== Overview
 
-==== Secure and Verified Boot
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "Generic vertically integrated system"]
+image::img_ch4_priv.png[]
+
+A generic vertically integrated system can be either virtualized or non-virtualized. 
+
+M-mode hosts a FW RoT. An OS or a Hypervisor in S or HS mode controls applications or guests. Guests and applications execute in U or VS/VU modes and trust the OS or Hypervisor to provide isolation guarantees.
```

Comment:
```suggestion
M-mode hosts a FW RoT that typically is bootstrapped by a HW RoT. An OS or a Hypervisor in S or HS mode controls applications or guest VMs. Guest VMs and applications execute in U and/or VS/VU modes and trust the OS or Hypervisor to provide isolation guarantees.
```

Reply:
The relationships between FW RoT and HW RoT are defined in the reference model. We are not assuming that *all* use cases have a HW RoT, or even need one. Obviously many do and should, but not all require one.

Reply:
"Guest" is also used as a general term. "Guest VM" is only one possible example. Others include "Guest TEE", "Confidential guest", etc., depending on use case.